src/coverlet.msbuild.tasks/CoverageResultTask.cs

```diff
@@ -54,11 +55,16 @@ public string ThresholdStat
             set { _thresholdStat = value; }
         }
 
+        public CoverageResultTask()
+        {
+            _logger = new MSBuildLogger(Log);
+        }
+
         public override bool Execute()
         {
             try
             {
-                Console.WriteLine("\nCalculating coverage result...");
+                _logger.LogInformation("Calculating coverage result...");
```

Comment:
As I said for the moment I would revert this to old `Console.WriteLine`

Reply:
if LogInformation's implementation is Console.WriteLine then it would be the same as the previous behavior

Reply:
Ok understood your point, personally I wouldn't mix infrastructure vs non infrastructure(mandatory console output) in same "interface".
I expect that when I'll pass v:quiet we won't show anything from this interface calls, but in this case we won't show for all but LogInformation usage.
I prefer another interface `IConsole.WriteLine` if we want make it testable and pluggable.
BTW this is only my opinion wait for @petli @tonerdo one.